.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install --upgrade pip
-          pip install -r requirements.txt -r requirements-dev.txt
+          pip install . .[dev]
```

Comment:
dont duplicate installation, you only need to specify .[dev], mandatory dependencies will be installed as well